packages/app/providers/Providers.tsx

```diff
@@ -17,7 +17,7 @@ import {
 export const Providers = ({ children }: PropsWithChildren) => {
   return (
     <WagmiConfig config={config}>
-      <ConnectKitProvider mode="light">
+      <ConnectKitProvider mode="light" options={{ initialChainId: 0 }}>
```

Comment:
Why `0` as initial chain ID?

Reply:
good question!

we do that so if we are on ethereum network on our wallet when connecting to stackly it keeps ethereum. Previously, it was triggering the wallet to change to gnosis and then again to change back to ethereum.

<img width="828" alt="image" src="https://github.com/SwaprHQ/stackly-ui/assets/5664434/0c97ddde-9d9c-45c4-bf4e-6289dfea6636">

https://docs.family.co/connectkit/api-reference#initialChainId

Reply:
Ahhhhhhh gotcha gotcha, makes a lot of sense now